roles/convert/tasks/main.yml

```diff
@@ -1,3 +1,19 @@
 ---
 # tasks file for convert
+
+- name: Convert2RHEL convert
+  ansible.builtin.shell: >
+    export PATH={{ convert_os_path }}
+    set -o pipefail;
+    convert2rhel convert
+    {{ convert_convert2rhel_opts }}
+    {{ convert_convert2rhel_enable_repos_args }}
+    {{ convert_convert2rhel_disable_repos_args }}
+    2>&1
+  register: convert2rhel
+  args:
+    executable: /bin/bash
+  async: "{{ convert_async_timeout_maximum | int }}"
+  poll: "{{ convert_async_poll_interval | int }}"
+  failed_when: convert2rhel.stderr | length > 0
```

Comment:
Oh, you are right, I didn't notice this :sweat_smile: Should it not be redirected then?

Reply:
I am not sure if it is on purpose, but leapp does have the redirect because it is streamed to log file and terminal (which is not in our case), so maybe in this case we could just check the return code of the command and fail then?

Reply:
I think @andywaltlova solution might be simpler.

Reply:
Okay, I changed it to check the return code.